ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/cygwin/tasks/main.yml

```diff
@@ -15,35 +15,19 @@
   register: cygwin_installed
   tags: cygwin
 
-# For details about the cache, see: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1210#issuecomment-679076545
-- name: Retrieve cygwin cache from ci.adoptopenjdk.net
+- name: Retrieve Cygwin setup
   win_get_url:
-    url: https://ci.adoptopenjdk.net/userContent/winansible/cygwin64.tar.gz
-    dest: C:/temp/cygwin64.tar.gz
-    checksum: 8f8b1d7d00233e384e725390c4018941fe7125290b94ecc0eeb3cf6684b4ce23
-    checksum_algorithm: sha256
-  when: (not cygwin_installed.stat.exists)
-  tags: cygwin
-
-- name: Retrieve cygwin toolset from ci.adoptopenjdk.net
-  win_get_url:
-    url: https://ci.adoptopenjdk.net/userContent/winansible/cygwin64_bootstrap.zip
-    dest: C:/temp/cyg_toolset.zip
-    checksum: d7950df964a0618ac47561d349276d4644e27f5b8f0a5fba9b10db702d09ecad
-    checksum_algorithm: sha256
-  when: (not cygwin_installed.stat.exists)
+    url: https://cygwin.com/setup-x86_64.exe
+    dest: C:\temp\cygwin.exe
```

Comment:
should we cleanup this file or does that happen somewhere else?

Reply:
We've historically always kept it on the machine, possibly to simplify manual updates? I don't think it necessarily needs to be on there though

Reply:
We'll leave it for now. Perhaps another PR to cleanup any tmp files

Reply:
I intend to have one of those fairly soon anyway :-)